org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/translator/BoundsComputer.java

```diff
@@ -198,6 +254,15 @@ private Expression allocateSubsetSig(SubsetSig sig) throws Err {
             return sum;
         }
         // Allocate a relation for this subset sig, then bound it
+        String label = (sig.label.startsWith("this/") ?
+                sig.label.substring(5):
+                    sig.label)+"%";
```

Comment:
this is a very strange formatting of the ternary operator.  I suggest putting everything in a single line (and adding spaces around both `?` and `:`

Reply:
fixed.